daphne/src/testing.rs

```diff
@@ -24,11 +24,19 @@ use std::{
 };
 use url::Url;
 
+// Secret key of "collector_hpke_config":
+// 60890f1e438bf1f0e9ad2bd839acf1341137eee623bf7906972bf1cc80bb5d7b
 pub const TASK_LIST: &str = r#"{
     "f285be3caf948fcfc36b7d32181c14db95c55f04f55a2db2ee439c5879264e1f": {
         "leader_url": "https://leader.biz/leadver/v1/",
         "helper_url": "http://helper.com:8788",
-        "collector_hpke_config": "f40020000100010020a761d90c8c76d3d76349a3794a439a1572ab1fb8f13531d69744c92ea7757d7f",
+        "collector_hpke_config": {
+            "id": 23,
+            "kem_id": "X25519HkdfSha256",
+            "kdf_id": "HkdfSha256",
+            "aead_id": "Aes128Gcm",
```

Comment:
@armfazh let me konw if you want to make this snake case in this PR.

Reply:
whatever JSON standard recommends

Reply:
Folks seem split: https://stackoverflow.com/questions/5543490/json-naming-convention-snake-case-camelcase-or-pascalcase

I think the main thing is to be consistent. I'll raise an issue for this.